database_admin/schema/create_schema.sql

```diff
@@ -159,6 +156,7 @@ BEGIN
     FROM system_advisories sa
     WHERE sa.system_id = NEW.id
       AND sa.when_patched IS NULL
+      AND change > 0
```

Comment:
Just to be sure, isn't it be `change != 0`?

Reply:
No, this is a logical error. It should never happen, but in the old code we performed insertion only when incrementing the counters.